aws-postgresql.yml

```diff
@@ -176,10 +176,12 @@ provision:
     type: boolean
     details: Allow minor version upgrades automatically during the maintenance window
     default: true
-  - field_name: maintenance_day
+  - &nullable_list
```

Comment:
- This looks to me like a string can that be set to null. I would call the anchor `nullable_string`, but it's called `nullable_list`. Have I missed something, or is there a typo?
- The issue with making the whole node an anchor is that certain fields _always_ need to be overridden when referencing the anchor. This makes it subject to subtle bugs in the future if someone were to add a new property to `maintenance_day` that should not be copied to all the references. I was thinking it would be better to define and inject the anchor by doing something like:
```yaml
  - <<: &nullable_string
      type: string
      nullable: true
    field_name: maintenance_day
    default: Sun
```
I'm no YAML expert, so I don't know if that works.